FactoryDesign/FactoryDesign/ActivityProduct.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FactoryDesign
+{
+    class ActivityProduct:IProduct
+    {
+        public string type = "Activity";
+        public string name = "Sky Diving";
```

Comment:
const variable should be defined as constant and ideally these should be private